File: app/src/main/java/io/euphoria/xkcd/app/impl/ui/UserListView.java
package io.euphoria.xkcd.app.impl.ui;

import android.content.Context;
import android.support.annotation.Nullable;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;
import android.util.AttributeSet;

/** Created by Xyzzy on 2020-03-30. */

public class UserListView extends RecyclerView {

    public UserListView(Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
        setLayoutManager(new LinearLayoutManager(context));
    }

}

File: app/src/main/java/io/euphoria/xkcd/app/impl/ui/views/UserListView.kt
package io.euphoria.xkcd.app.impl.ui.views

import android.content.Context
import android.util.AttributeSet
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView

/** Created by Xyzzy on 2020-03-30.  */
class UserListView(
    context: Context,
    attrs: AttributeSet?
) : RecyclerView(context, attrs) {
    init {
        layoutManager = LinearLayoutManager(context)
    }
}
